src/scss/_home.scss

```diff
@@ -0,0 +1,65 @@
+.Header-Container {
+  padding: 10px 20px;
+}
+
+.executive-details {
+  background-color: #5788e4;
+  padding: 0 20px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.executive-details1 {
+  background-color: #5788e4;
+  padding: 2px 10px 2px 0;
+  width: 70%;
+}
+
+.Filter {
+  border-radius: 6px;
+  border: none;
+  color: black;
+  width: 30%;
+
+  option {
+    color: black;
+  }
+}
+
+.home {
+  display: grid;
+  grid-template-columns: 50% 50%;
+  //   grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
```

Comment:
- [ ] Kindly remove this commented code, you can use GitHistory to refer to previous code versions.